templates/feed-stylesheet.xsl

```diff
@@ -1,14 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
-<xsl:stylesheet version="2.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
-    <xsl:output method="html" version="1.0" encoding="UTF-8" indent="yes"/>
+<xsl:stylesheet version="2.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform" xmlns:itunes="http://www.itunes.com/dtds/podcast-1.0.dtd">
+    <xsl:output method="html" version="1.0" encoding="ISO-8859-1" indent="yes"/>
     <xsl:template match="/">
+
         <html xmlns="http://www.w3.org/1999/xhtml">
-            <head>
-                <title><xsl:value-of select="/rss/channel/title"/> RSS Feed</title>
+            <head>            	
+                <title><xsl:value-of select="/rss/channel/title"/> - RSS Feed</title>
                 <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
                 <style type="text/css">
+                	@import url('https://fonts.googleapis.com/css?family=Montserrat:300,400,500,600');
+
                     body {
-                        font-family: Helvetica, Arial, sans-serif;
+                        font-family: 'Montserrat', sans-serif;
```

Comment:
I'm curious, what made you change this font? As far as I know Montserrat is not a common system font, so to maintain compatibility with the vast majority of browsers/operating systems, it would be preferable to stick to a Helvetica, Arial, sans-serif